x-pack/winlogbeat/module/security/ingest/security.yml

```diff
@@ -3442,6 +3458,51 @@ processors:
           splitSidList(ctx.winlog.event_data.SidList, params, ctx);
         }
 
+  - set:
```

Comment:
Note for ref: Code from here https://github.com/elastic/integrations/blob/4d554d3561b2e5ccdbc23c2f2f82fb95d731219a/packages/system/data_stream/security/elasticsearch/ingest_pipeline/standard.yml#L3376